Sources/Comet/Helpers/Logging/DataTaskPublisher+Debug.swift

```diff
@@ -0,0 +1,64 @@
+//
+//  DataTaskPublisher+Debug.swift
+//  Comet
+//
+//  Created by Jaroslav Novák on 17.01.2022.
+//  Copyright © 2022 Etnetera. All rights reserved.
+//
+
+import Combine
+import Foundation
+
+public extension URLSession.DataTaskPublisher {
+    func debug(
+        logLevel: RequestLogLevel,
+        logger: @escaping (String) -> Void
+    ) -> Publishers.HandleEvents<Self> {
+        handleEvents(
+            receiveSubscription: { _ in
+                if logLevel.contains(.url) {
```

Comment:
Maybe I would use the condition to .none and then did not perform other operations.

Reply:
Added guard that checks, if log level contains `.none` equals false, then continue with code below. Otherwise return.